trade-finance-manager-api/api-tests/v1/deals/deals-submit.second-submission.api-test.js

```diff
@@ -44,6 +45,10 @@ jest.mock('../../../src/v1/controllers/deal.controller', () => ({
   submitACBSIfAllPartiesHaveUrn: jest.fn(),
 }));
 
+const updateGefActivitySpy = jest.fn(() => Promise.resolve(
+  { ...MOCK_GEF_DEAL_MIN },
+));
```

Comment:
Could just be `Promise.resolve(MOCK_GEF_DEAL_MIN);`

Reply:
done